skt/kernelbuilder.py

```diff
@@ -327,13 +336,53 @@ def make_rpm_repo(self, rpm_files):
 
         return repo_dir
 
+    def find_extras(self, rel_path, callback=None, followlinks=True):
+        """
+        Find files to be distributed alongside with the kernel.
+
+        Args:
+            rel_path:    A path (relative to the kernel tree root) in which
+                         extras are sought (recursively).
+
+            callback:    If given, serves as a result pruning function for
+                         filter.
+
+            followlinks: Follow symbolic links to subdirectories on systems
+                         that support them. Set this to True only if you're
+                         sure that this won't cause loops.
+
+        Returns:
+            A (possibly pruned) list of paths recursively found in
+            `rel_path`. Returns an empty list if no matches were found
+            or an exception occured during pruning.
+        """
+        source_dir_abs = os.path.abspath(self.source_dir)
+        target_dir = source_dir_abs + rel_path
```

Comment:
Could you please use `os.path.join` or `skt.misc.join_with_slash` for path joins?

I also see that when you call the method you use a starting slash with the directory path which makes the path look like absolute path which is confusing. Using one of the mentioned functions would help with that too.